README.md

```diff
@@ -1 +1,35 @@
-# Hello-world
\ No newline at end of file
+Hello Microverse
+The project is about Microverse courses.
+
+Built With
+Javascript, CSS, HTML
+
+Prerequisites
+Setup
+Install
+Usage
+Run tests
+Deployment
+Authors
+👤 Author1
+
+GitHub: @Victoria129
+Twitter: @Tumwebaze129
+LinkedIn: victoria-tumwebaze-01701424a
+
+🤝 Contributing
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+Show your support
+Give a ⭐️ if you like this project!
+
+Acknowledgments
+Hat tip to anyone whose code was used
+Inspiration
+etc
+📝 License
+This project is MIT licensed.
```

Comment:
Kindly note that currently this links to nothing. This should point to an licence file named MIT.md file.
The instructions to set this up are contained at the license section of the [readme template](https://github.com/microverseinc/readme-template/blob/master/README.md)
<img width="1028" alt="Screenshot 2022-09-21 at 02 43 49" src="https://user-images.githubusercontent.com/48546992/191383910-1e92a826-843d-4fb3-9454-e3c4171d9924.png">